docs/howto/windows.txt

```diff
@@ -16,52 +19,64 @@ Install Python
 Django is a Python web framework, thus requiring Python to be installed on your
 machine.
 
-To install Python on your machine go to https://python.org/download/, and
-download a Windows MSI installer for Python. Once downloaded, run the MSI
-installer and follow the on-screen instructions.
+To install Python on your machine go to https://python.org/downloads/. The web
+site should detect that you're on Windows, and offer you a download button for
+the latest Python version. Download the EXE file and run it.
```

Comment:
can we use the same terminology as the Python download page, e.g. "executable installer" instead of "EXE file"?